doc/nmstatectl.8.in

```diff
@@ -0,0 +1,91 @@
+.\" Manpage for nmstatectl.
+.TH nmstatectl 8 "@DATE@" "@VERSION@" "nmstatectl man page"
+.SH NAME
+nmstatectl \- A nmstate command line tool
+.SH SYNOPSIS
+.B nmstatectl show \fR[<\fIinterface_name\fR>] [\fB--json\fR]
+.br
+.B nmstatectl set \fR<\fIstate_file_path\fR> [\fIOPTIONS\fR]
+.br
+.B nmstatectl edit \fR[<\fIinterface_names\fR>] [\fIOPTIONS\fR]
+.br
+.B nmstatectl rollback [\fR<\fIcheckpoint_path\fR>]
+.br
+.B nmstatectl commit [\fR<\fIcheckpoint_path\fR>]
+.SH DESCRIPTION
+.B nmstatectl\fR is created for users who want to try out nmstate without using
+\fIlibnmstate\fR.
+.PP
+.B show
+.RS
+Query the current network state. \fIYAML\fR is the default output format. Use
+the \fB--json\fR argument to change the output format to \fIJSON\fR. To limit
+the output state to include certain interfaces only, please specify the
+interface name. Please be advised, global config like DNS will be included.
+.PP
+For multiple interface names, use comma to separate them. You can also use
+patterns for interface names:
+.RS
+.B *\fR matches everything
+.br
+.B ?\fR matches any single character
+.br
+.B [seq]\fR matches any character in seq
+.br
+.B [!seq]\fR matches any character not in seq
```

Comment:
Please provide a example as `cli_guide` did:

For example, to show all interfaces starts with eth:
```
nmstatectl show eth\*
# The backslash is required to stop shell expanding '*' to file names.
```